### ISSUE TITLE
InteractionReceiver NRE OnDisable()

### ISSUE THREAD

**Issue by eluchsinger** [Microsoft/MixedRealityToolkit-Unity#1384]
## Overview
I am getting a `NullReferenceException` when I exit my application in Editor mode. The NRE comes from the [InteractionReceiver](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/UX/Scripts/Receivers/InteractionReceiver.cs).

It is possible that it is a bug on my side, but the `InteractionReceiver` class does not check for null. In my opinion, this shouldn't be possible because the part of the code is closed in itself, I don't see it referencing a part of my project (except that it is assigned to a GameObject as a component).

## Expected Behavior
Be able to stop the editor from running without getting an exception.

## Actual Behavior
After stopping the execution in the editor:
```
NullReferenceException: Object reference not set to an instance of an object
HoloToolkit.Unity.Receivers.InteractionReceiver.OnDisable () (at Assets/HoloToolkit/UX/Scripts/Receivers/InteractionReceiver.cs:73)
```

## Unity Editor Version
2017.2.0f3

## Mixed Reality Toolkit Release Version
Last master https://github.com/Microsoft/MixedRealityToolkit-Unity/commit/272d5fd10c0b836a2a7bcb021a655dcf2288d7e9

**Comment by paseb** [comment]
I'll have a fix for this.  I had fixed it in a separate branch and forgot to integrate it since it was just a null ref in editor alone with a race condition with references to destroyed objects on disable.

**Comment by keveleigh** [comment]
I believe I have this fixed in #1339 by checking `IsInitialized` before making those calls.

**Comment by keveleigh** [comment]
Looks like GitHub didn't automatically close these issues when merged into master. I'll close these manually and look into why.
